src/Bundle/test/src/Subscription/Entity/Subscription.php

```diff
@@ -33,25 +33,25 @@
 #[Index(grid: 'app_subscription')]
 #[Create]
 #[Update]
+#[BulkDelete]
 #[Delete]
 #[Show(template: 'subscription/show.html.twig')]
 #[ORM\Entity]
 class Subscription implements ResourceInterface
 {
     public function __construct(
         #[ORM\Id]
-        #[ORM\Column(type: 'uuid', unique: true)]
-        #[ORM\GeneratedValue(strategy: 'CUSTOM')]
-        #[ORM\CustomIdGenerator(class: 'doctrine.uuid_generator')]
-        public ?Uuid $id = null,
+        #[ORM\Column(type: 'integer', unique: true)]
+        #[ORM\GeneratedValue(strategy: 'AUTO')]
+        public ?int $id = null,
         #[Assert\NotBlank]
         #[Assert\Email]
         #[ORM\Column(name: 'name', nullable: false)]
         public ?string $email = null,
     ) {
     }
 
-    public function getId(): ?Uuid
+    public function getId(): ?int
```

Comment:
I've simplified this cause I had some troubles. But it works on a test project with an uuid on a mysql database.

Reply:
What were the troubles?